src/Validator/Constraints/InlineConstraint.php

```diff
@@ -51,6 +51,9 @@ public function __construct($options = null)
 
     public function __sleep(): array
     {
+        // Initialize "groups" option if it is not set see parent:: __sleep();
+        $this->groups = $this->groups ?? [];
```

Comment:
IMHO, we shouldn't change what is mentioned in this comment: https://github.com/symfony/symfony/issues/45042#issuecomment-1048864055

Reply:
Would you agree, if I do this:

```
       // @phpstan-ignore-next-line to, initialize "groups" option if it is not set
        $this->groups;
```

Reply:
On my side yes, do you agree @VincentLanglet ?